language/stdlib/modules/libra_account.move

```diff
@@ -81,22 +81,22 @@ module LibraAccount {
         // Total number of events emitted to this event stream.
         counter: u64,
         // A globally unique ID for this event stream.
-        guid: bytearray,
+        guid: vector<u8>,
     }
 
     // Deposits the `to_deposit` coin into the `payee`'s account
     public fun deposit(payee: address, to_deposit: LibraCoin::T) acquires T {
-        // Since we don't have bytearray literals in the source language at
+        // Since we don't have vector<u8> literals in the source language at
         // the moment.
-        // FIXME: Update this once we have bytearray literals
+        // FIXME: Update this once we have vector<u8> literals
```

Comment:
Perhaps use `Vector::empty<u8>`, but keep the TODO? An empty vector literal might be better once we have them, but `empty` seems better than `u64_to_bytes`.